visidata/deprecated.py

```diff
@@ -1,16 +1,20 @@
-from visidata import VisiData
+from visidata import Column, PyobjSheet, VisiData, vd
 import visidata
 
 alias = visidata.BaseSheet.bindkey
 
 def deprecated(ver, instead=''):
     def decorator(func):
         def wrapper(*args, **kwargs):
-            # ideally would include a stacktrace
+            import traceback
+
+            for line in reversed(traceback.extract_stack(limit=6)[:-1]):
+                vd.warning(f'    file {line.filename} at line {line.lineno} in {line.name}')
```

Comment:
What do you think about putting the entire traceback in a single multiline string?

Reply:
I'd like that if you could help me sort out how. I tried a couple ways of sending a multi-line string to `vd.warning`. The message history seemed to have trouble with `\t` or `\n` characters:

<img width="1121" alt="image" src="https://user-images.githubusercontent.com/19539955/96085031-298a3380-0e8e-11eb-9aa7-ec1b4ea52053.png">

If there's a nice way to avoid calling `vd.warning` repeatedly I'm all for it.

Reply:
If you go multiline with `v`, the newlines are handled better (not sure about the tabs or overall workability though).  If we can get this to work reasonably, I prefer it somewhat, but if it's too much of a pain, we can stick with the original stream of vd.warning.

Actually, now that I think about it, in the deprecated usage case, do we actually want the full stack trace, or just the file/line where the deprecated function is used directly?